src/react-app-env.d.ts

```diff
@@ -1 +1,5 @@
 /// <reference types="react-scripts" />
+declare module '*.mp4' {
```

Comment:
Dette var visst nødvendig for å få create react app til å tillate å laste inn videofiler.